yt_dlp/extractor/floatplane.py

```diff
@@ -0,0 +1,155 @@
+from .common import InfoExtractor
+
+from ..utils import (
+    ExtractorError,
+    parse_iso8601,
+    determine_ext,
+    unified_strdate,
+    traverse_obj
+)
+
+
+class FloatplaneIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://(?:www\.)?floatplane\.com/post/(?P<id>[a-zA-Z0-9-]+))'
+    _TESTS = [{
+        'url': 'https://www.floatplane.com/post/DdShecU983',
+        'info_dict': {
+               'id': 'DdShecU983',
+               'ext': 'mp4',
+               'channel': 'LinusTechTips',
+               'title': 'TQ: Are Intel Arc Graphics A Bad Idea?',
+               'description': '<p><br /></p>',
```

Comment:
this is obv not the correct description...

Reply:
I could use some guidance here.  The descriptions given are rich HTML blobs which can contain all sorts of HTML tags.  Any idea how other platforms handle this?

Reply:
> Use `clean_html`

I assume u didn't see my edit